packages/core/upload/admin/src/hooks/useEditAsset.js

```diff
@@ -21,7 +21,7 @@ const editAssetRequest = (asset, file, cancelToken, onProgress, post) => {
     JSON.stringify({
       alternativeText: asset.alternativeText,
       caption: asset.caption,
-      folder: asset.folder?.id,
+      folder: asset.folder,
```

Comment:
The update rectifies an oversight, where `asset.folder` was utilized as an object instead of its intended use as a numeric value.